doc/dev/GETTING_STARTED.md

```diff
@@ -19,7 +19,7 @@ should work for other OSes as stated, as long as listed packages are available.
 ### Install dependencies
 
 First make sure your system is up-to-date,
-then install the required dependencies using the terminal.
+then install the required dependencies (nlohmann-json, hdf5, netcdf, ospray, fmt and verdict) using the terminal.
```

Comment:
please list the actual package in the below commands instead. If you dont know for a specific os, that's ok.

Please list only the non-optional dependencies. eg: ospray should not be needed

Reply:
Right, the additional dependencies are only required for Arch and its derivatives. They were not required to be installed for Ubuntu 23. I have modified the docs appropriately.